lib/airbrake-ruby/backtrace.rb

```diff
@@ -188,6 +193,13 @@ def match_frame(regexp, stackframe)
 
         Patterns::GENERIC.match(stackframe)
       end
+
+      def read_code_hunk(config, frame)
+        code_hunk = Airbrake::CodeHunk.new(config, frame[:file], frame[:line]).to_h
```

Comment:
Is there a good reason the method is called `to_h`? I see none.

Reply:
`read`?

Reply:
I would expect `CodeHunk` to be shared and code look like this:

```
def constructor
  @code_hunk = Airbrake::CodeHunk.new(config)

def populate_code(frame)
  @code = @code_hunk.get(file, line)
  if @code
    frame[:code] = @code
```

Reply:
No big deal, I can init at the same place.